src/components/ResolutionEditButton.js

```diff
@@ -1,59 +1,63 @@
-import React, { Component } from 'react';
+import React, { useState, useRef } from 'react';
 import PropTypes from 'prop-types';
 
-import { Wizard } from '@redhat-cloud-services/frontend-components/Wizard';
+import { Button, Modal, ModalVariant } from '@patternfly/react-core';
 import ResolutionStep from './ResolutionModal/ResolutionStep';
 
-class ResolutionEditButton extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      open: false,
-    };
-  }
+const ResolutionEditButton = ({
+  remediation,
+  issue,
+  onResolutionSelected,
+  getResolutions,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const openModal = () => setIsOpen(true);
```

Comment:
I don't think we need to have seperate constant for this, we can directly call `setIsOpen(true)` on line 29.